public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>React App - Contact list agge</title>
```

Comment:
This is Testing Comment

Reply:
comment 1

Reply:
updated as per comment added on index file